@Edf2Mat/Edf2Mat.m

```diff
@@ -274,9 +274,9 @@
               ['File ' filename ' not found!']);
             
             obj.filename = filename;
+            [nowine, ~] = system('wine --version');
             
-            %britt added
-            if(isunix)
+            if isunix && ~ismac
```

Comment:
instead of putting logic into the if statements, it would be more readable to just rewrite the `isunix` into one expression e.g. `isunix = strcmp(computer(), 'maci64')`

Reply:
That's kind of the problem here. Afaik here is no `islinux` test yet.
And because there are no linux mex files we have to test for unix but not mac.

Mathworks kind of recommends the same thing: https://de.mathworks.com/help/matlab/ref/isunix.html

Reply:
sorry, my fault, copy and pasted the test on my mac!
What I meant was of course:
```
isunix = strcmp(computer(), 'glnxa64')
```
or even if you want to have the word linux in it ;)

```
islinux = strcmp(computer(), 'glnxa64')
```
so it's just rewriting the function call to isunix() to a defined variable containing if it is linux

Reply:
can I safely ignore 32-bit?

Reply:
good point, for linux I'd say so...

Reply:
I'll anyway put out the source of the mex file as soon as I find time ... and the everyone can compile it to his own needs

Reply:
I guess `strfind(computer, 'GLNX')` is just as readable